x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -169,17 +170,17 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         http: core.http,
       });
 
-      const filterManager = new FilterManager(core.uiSettings);
-
       // used for creating a custom stateful KQL Query Bar
       const customDataService: DataPublicPluginStart = {
         ...startPlugins.data,
-        query: {
-          ...query,
-          filterManager,
-        },
+        query,
+        // @ts-expect-error
```

Comment:
Why not just update the `DataPublicPluginStart` type for this to avoid the `@ts-expect-error`?

Reply:
Actually, I just added it as an aid to make sure the correct service is being passed. It has no meaning so I will remove it.